services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -424,7 +424,7 @@ public static Builder builder() {
     private Point destination;
     private Point origin;
     private String[] approaches;
-    private Integer[] waypoints;
```

Comment:
This was correct as-was. Should be `waypoints` as one word, no capital P

Reply:
Yeah, you're correct. Will update in a follow-up PR. Thanks for flagging 🙇